cups/debug.c

```diff
@@ -485,7 +485,7 @@ _cups_safe_vsnprintf(
 	case 'e' :
 	case 'f' :
 	case 'g' :
-	    if ((size_t)(width + 2) > sizeof(temp))
+	    if ((size_t)(width) > sizeof(temp) - 2)
```

Comment:
I really prefer the original code - compare the width + 2 against the size of the buffer, not the width with the size less 2. And regardless I'd want parenthesis (necessary or not) around the calculation.